File: PHP/1007.php
<?php
    $a = trim(fgets(STDIN));
    $b = trim(fgets(STDIN));
    $c = trim(fgets(STDIN));
    $d = trim(fgets(STDIN));
    
    printf("DIFERENCA = %d\n", $a * $b - $c * $d);
?>

File: Dart/1007.dart
import 'dart:io';

void main() {
  int a = int.parse(stdin.readLineSync());
  int b = int.parse(stdin.readLineSync());
  int c = int.parse(stdin.readLineSync());
  int d = int.parse(stdin.readLineSync());
  
  int r = a * b - c * d;
  print("DIFERENCA = $r");
}